media/injectScript.js

```diff
@@ -83,6 +83,7 @@ function createConsoleOverride(type) {
 
 		try {
 			stringifiedMsg = JSON.stringify(msg);
+			if (!stringifiedMsg) throw new Error('stringifiedMsg is undefined');

```

Comment:
`stringifiedMsg` is an internal implementation detail. Although this exception doesn't get surfaced to the top of the stack, maybe a more appropriate message would be "message is not in JSON format"?

Reply:
Thanks, that makes sense. Let me know if my newest commit works @andreamah